tests/test_log.py

```diff
@@ -0,0 +1,8 @@
+from vorta.log import LogTimedRotatingFileHandler
+
+
+def test_log_file_suffix():
+    """Ensure log files always have '.log' suffix."""
+
+    handler = LogTimedRotatingFileHandler('mylog.log', when='d', interval=1, backupCount=5)
+    assert handler.suffix.endswith('.log')
```

Comment:
guess that is not enough to test the dynamic behaviour (which is hard).
maybe rather have no automated test than a pointless one?

Reply:
@ThomasWaldmann  Agreed - I missed the mark here. 

Currently we are using a default filename format for the log (ex. vorta.log.[timestamp]). I am considering creating a function for `namer` that moves the '.log' to the end of filename (ex. vorta.[timestamp].log). 

To test this, I was considering testing the parts of the code that call `rotation_filename`, as this function uses `namer` to set the custom filenames. I'm curious what you think of this approach? What behavior would you recommend testing?

Reply:
Guess the code is now simple enough to not need a test.